lib/scrabble.rb

```diff
@@ -14,5 +14,56 @@ def point_values
       "U"=>1, "V"=>4, "W"=>4, "X"=>8,
       "Y"=>4, "Z"=>10
     }
+end
+
+# def boss_method(word)
+#   break_word_into_array(word)
+#
+#   score()
+# end
+
+def break_word_into_an_array(word)
+  unless word == "" || word == nil
+  word_array = word.upcase.chars #now I should have the array ["H", "E", "L", "L", "O"]
+end
+
+def get_individual_letters(word_array)
+  until word_array == []
+  score(word_array.pop) #until the array is empty, pop off letters and feed them into the letter-scoring method
   end
 end
+
+  def score(letter)
+    score = []
+    point_values.each do |letter|
+      if letter == point_values.key
+        score << point_values.value #compare each letter from the word array to the keys in point_values, and shovel those values into an a new array
+      end
+      word_score(score) #the score argument should be the array [1,2,1,1,1], for "hello"
+  end
+
+def word_score(score)
+  unless score == [] || score == nil
+    wordscore = score.inject do |sum, individual score| #this should add the values across the score array
```

Comment:
If `individual_score` is intended to be a single variable, the words need to be connected by an underscore.